docs/user/power/install_upi.md

```diff
@@ -0,0 +1,300 @@
+# Install: Power User Provided Infrastructure
+
+The steps for performing a UPI-based install are outlined here. Example automation is provided [here](#example-power-upi-configuration) to help model your own.
+
+## Table of contents
+
+1. Minimum compute requirements
```

Comment:
nit: it would be nice for users if these were links to the sections. I can't find official docs but this works: https://stackoverflow.com/questions/6695439/how-to-link-to-a-named-anchor-in-multimarkdown/15843220#15843220

Reply:
sure I'll add links to section.